src/backend/configs/caliopen.yaml.template

```diff
@@ -18,6 +18,14 @@ broker:
     port: 2525
     host: localhost
 
+object_store:
+    db_size_limit: 1048576
+    service: minio
```

Comment:
no explicit minio reference, it's an object store (s3 if you like) we don't care about implementation.